app/controllers/users_controller.rb

```diff
@@ -42,32 +40,28 @@ def change_password
 
   def update
     @user = current_user
-    @errors = []
-    @errors.push('first_name can not be empty') unless user_params[:first_name].present?
-    @errors.push('last_name can not be empty') unless user_params[:last_name].present?
-    @errors.push('phone can not be empty') unless user_params[:phone].present?
-    @errors.push('address can not be empty') unless user_params[:address].present?
-    if @user.update(update_params)
-      redirect_to root_path, notice: 'Account updated'
+    error = acceptable_image(params[:user][:picture],@user)
+    if error.size == 0
+      @user.picture.purge
+    end
+    if error.size == 0 && @user.update(update_params)
+      redirect_to user_update_path, notice: 'Account updated'
     else
-      flash[:error] = 'Please try again'
+      flash[:notice] = 'Please try again'
       render 'edit', status: :unprocessable_entity
     end
   end
 
   def edit
-    @user = UserViewModel.new(
-      first_name: current_user.first_name,
-      last_name: current_user.last_name,
-      email: current_user.email,
-      phone: current_user.phone,
-      address: current_user.address
-    )
+    @user = current_user
   end
 
   def create
-    @user = User.new(user_params)
-    if @user.save
+    @organization = Organization.new(organization_params)
+    error = acceptable_image(params[:organization][:users_attributes]["0"][:picture],@organization.users.first)
+    if error.size ==0 && @organization.save
+      current_email = params[:organization][:users_attributes]["0"][:email]
```

Comment:
I'm bit confuse about the use of ["0"] mentioned here. Can you please explain? Thank you!